evap/rewards/views.py

```diff
@@ -122,5 +122,7 @@ def semester_activation(request, semester_id, active):
     active = active == 'on'
 
     SemesterActivation.objects.update_or_create(semester=semester, defaults={'is_active': active})
+    if active:
+        grant_eligible_reward_points_for_semester(request, semester)
```

Comment:
We agreed on adding a confirmation modal (because this is irreversible from the UI). Modal should also contain a note that the request will take a while to process.

Reply:
done